Dart/src/com/jetbrains/lang/dart/analyzer/DartAnalysisServerService.java

```diff
@@ -960,6 +968,80 @@ public void onError(final RequestError error) {
     return results;
   }
 
+  public boolean edit_isPostfixCompletionApplicable(VirtualFile file, int offset, String key) {
+    final AnalysisServer server = myServer;
+    if (server == null) {
+      return false;
+    }
+    final String filePath = FileUtil.toSystemDependentName(file.getPath());
+    final Ref<Boolean> resultRef = Ref.create();
+    final CountDownLatch latch = new CountDownLatch(1);
+    server.edit_isPostfixCompletionApplicable(filePath, key, offset, new IsPostfixCompletionApplicableConsumer() {
+      @Override
+      public void isPostfixCompletionApplicable(Boolean value) {
+        resultRef.set(value);
+        latch.countDown();
+      }
+    });
+
+    awaitForLatchCheckingCanceled(server, latch, POSTFIX_COMPLETION_TIMEOUT);
+    return resultRef.get();
```

Comment:
This may cause NPE because of unboxing. Fixed by initial `false` value for `Ref.create()`